packages/reports/addon/templates/components/missing-intervals-warning.hbs

```diff
@@ -1,30 +1,32 @@
-{{#liquid-if warningEnabled class="missing-intervals-warning"}}
-  <div class="missing-intervals-warning__contents" role="button" onclick={{action (pipe (action (toggle "showDetails" this)) (action onDetailsToggle))}}>
-    {{navi-icon "exclamation-triangle" class="missing-intervals-warning__warning-icon"}}
-    <span class="missing-intervals-warning__message-text">Results have missing data.</span>
-    <span class="missing-intervals-warning__details-trigger">
-      {{#if showDetails}}
-        <span class="missing-intervals-warning__details-desc">Click to close.</span>
-        {{navi-icon "chevron-down" class="missing-intervals-warning__arrow-icon"}}
-      {{else}}
-        <span class="missing-intervals-warning__details-desc">Click for more details.</span>
-        {{navi-icon "chevron-up" class="missing-intervals-warning__arrow-icon"}}
-      {{/if}}
-    </span>
-  </div>
-  {{#liquid-if showDetails class="missing-intervals-warning__details"}}
-    <div class="missing-intervals-warning__details-content">
-      <div class="missing-intervals-warning__details-help-text">The following intervals from the result set have missing data:</div>
-      <div class="missing-intervals-warning__dates">
-        {{#each missingIntervals as |interval|}}
-          <div class="missing-intervals-warning__date-row">
-            {{navi-icon "times" class="missing-intervals-warning__x-icon"}}
-            <span class="missing-intervals-warning__date-interval">{{interval}}</span>
-          </div>
-        {{/each}}
-      </div>
-      <div class="missing-intervals-warning__disclaimer">Note: Listed intervals include both the start and end dates.</div>
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<AnimatedContainer class="missing-intervals-warning" @motion={{this.resizeMotion}}>
+  {{#animated-if this.warningEnabled initialInsertion=true use=this.drawerTransition}}
+    <div class="missing-intervals-warning__contents" role="button" {{on "click" (toggle "showDetails" this)}}>
+      <NaviIcon @icon="exclamation-triangle" class="missing-intervals-warning__warning-icon" />
+      <span class="missing-intervals-warning__message-text">Results have missing data.</span>
+      <span class="missing-intervals-warning__details-trigger">
+        {{#animated-if this.showDetails use=this.fadeTransition}}
+          <span class="missing-intervals-warning__details-desc">Click to close.</span>
+          <NaviIcon @icon="chevron-down" class="missing-intervals-warning__arrow-icon" />
+        {{else}}
+          <span class="missing-intervals-warning__details-desc">Click for more details.</span>
+          <NaviIcon @icon="chevron-up" class="missing-intervals-warning__arrow-icon" />
+        {{/animated-if}}
+      </span>
     </div>
-  {{/liquid-if}}
-{{/liquid-if}}
+    {{#if this.showDetails}}
```

Comment:
Why isn't the details expansion animated anymore?

Reply:
It is, actually. The `<AnimationContainer>` will animate content resizes.

Reply:
I see. So we're only using another `animated-if` on the `Click for more details` section because it's not actually resizing the content?